doc/parameters/param_list.md

```diff
@@ -155,8 +155,11 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | ----------| --------| ------------|
 | controller_frequency | 20.0 | Frequency to run controller |
 | progress_checker_plugin | "progress_checker" | Check the progress of the robot |
+| progress_checker.plugin | "nav2_controller::SimpleProgressChecker" | Default plugin |
 | goal_checker_plugin | "goal_checker" | Check if the goal has been reached |
+| goal_checker.plugin | "nav2_controller::SimpleGoalChecker" | Default plugin |
 | controller_plugins | ["FollowPath"] | List of mapped names for controller plugins for processing requests and parameters |
+| FollowPath.plugin | "dwb_core::DWBLocalPlanner" | Default plugin |
```

Comment:
Same

Reply:
Should be `<controller id>.plugin`?

Reply:
Well this is telling the default ... Either way I don't think this PR is the right place to argue with decisions we made when making this MD file. @siddhya didn't edit this and I think the PR should go in with only changes from his contributions

Reply:
That's a fair point, I agree.